kotlin/.gitignore

```diff
@@ -1,3 +1,5 @@
 .gradle/
 out/
 gradle/
+build/
+local.properties
```

Comment:
Might also be useful to add `.idea/` as well

Reply:
Done!